activation/post_states.go

```diff
@@ -9,28 +9,28 @@ import (
 	"github.com/spacemeshos/go-spacemesh/common/types"
 )
 
-type postStates struct {
+type PostStatesWrapper struct {
```

Comment:
I'm not sure if `Wrapper` is a suitable name for this.

Reply:
I'm happy to accept any other suggestions. PostStates is already defined in this package. This wrapped is just adding locking.

Reply:
This is the service for post states. So we could call it `PostStatesService` but the name is redundant, that's why it is `postStates`.

Reply:
Also the type shouldn't need to be exported, why the change?

Reply:
I need it for a field in the state.